src/components/Header.js

```diff
@@ -10,11 +10,11 @@ class Header extends React.Component {
   render() {
     return (
       <header>
-        <h1 itemprop="name">{this.props.title}</h1>
-        <em itemprop="description">{this.props.baseline}</em>
+        <h1 itemProp="name">{this.props.title}</h1>
```

Comment:
itemprop != itemProp (same for itemscope)

Reply:
According to https://reactjs.org/docs/dom-elements.html, 

```
In React, all DOM properties and attributes (including event handlers) should be camelCased. For example, the HTML attribute tabindex corresponds to the attribute tabIndex in React. The exception is aria-* and data-* attributes, which should be lowercased. For example, you can keep aria-label as aria-label.
```

The attributes are lowercased on rendering.